.template/addons/phrase_app/spec/codebase/codebase_spec.rb

```diff
@@ -0,0 +1,21 @@
+insert_into_file 'spec/codebase/codebase_spec.rb', before: %r{end\Z} do
+  <<-EOT
```

Comment:
We always use `EOT` as heredoc but rubocop always cries for a [meaningful heredoc naming](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/HeredocDelimiterNaming). Also adding the magic comment `frozen_string_literal: true` at the start of these types of files. Should follow these conventions?

Reply:
Hi @rafayet-monon Currently in the template we didn't have Rubocop set up yet, and many files are using `EOT` right now, also for the `frozen_string_literal: true` only 1 file having that 😃 so we will following the current template codebase @rafayet-monon 

And all will fix when we add the rubucop to the template itself https://github.com/nimblehq/rails-templates/issues/168

![Uploading image.png…]()